src/mechanics/timeIntegration/TimeIntegrationBase.h

```diff
@@ -184,47 +169,13 @@ class TimeIntegrationBase
     //! @param if delete is set, all the content of the directory will be removed
     void SetResultDirectory(std::string rResultDir, bool rDelete = false);
 
-    //! @brief returns the result directory
-    std::string GetResultDirectory()const
-    {
-    	return mResultDir;
-    }
 
     //! @brief sets automatic time stepping (on or off)
     void SetAutomaticTimeStepping(bool rAutomaticTimeStepping)
     {
     	mAutomaticTimeStepping = rAutomaticTimeStepping;
     }
 
-    //! @brief returns if automatic time stepping is turned on
-    bool GetAutomaticTimeStepping()const
-    {
-    	return mAutomaticTimeStepping;
-    }
-
-    //! @brief sets the coefficient matrix check (on or off)
-    void SetCheckCoefficientMatrix(bool rCheckCoefficientMatrix)
-    {
-        mCheckCoefficientMatrix = rCheckCoefficientMatrix;
```

Comment:
If you remove both getter and setter for a member, you can remove the member as well, right?
mCheckCoefficientMatrix and mExportDataFileNodes